homeassistant/components/glances/__init__.py

```diff
@@ -3,76 +3,27 @@
 import logging
 
 from glances_api import Glances, exceptions
-import voluptuous as vol
 
-from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
     CONF_HOST,
     CONF_NAME,
-    CONF_PASSWORD,
-    CONF_PORT,
     CONF_SCAN_INTERVAL,
-    CONF_SSL,
-    CONF_USERNAME,
     CONF_VERIFY_SSL,
     Platform,
 )
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
-import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.dispatcher import async_dispatcher_send
 from homeassistant.helpers.event import async_track_time_interval
 from homeassistant.helpers.httpx_client import get_async_client
-from homeassistant.helpers.typing import ConfigType
-
-from .const import (
-    CONF_VERSION,
-    DATA_UPDATED,
-    DEFAULT_HOST,
-    DEFAULT_NAME,
-    DEFAULT_PORT,
-    DEFAULT_SCAN_INTERVAL,
-    DEFAULT_VERSION,
-    DOMAIN,
-)
+
+from .const import DATA_UPDATED, DEFAULT_SCAN_INTERVAL, DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 PLATFORMS = [Platform.SENSOR]
 
-GLANCES_SCHEMA = vol.All(
-    vol.Schema(
-        {
-            vol.Required(CONF_HOST, default=DEFAULT_HOST): cv.string,
-            vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-            vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-            vol.Optional(CONF_USERNAME): cv.string,
-            vol.Optional(CONF_PASSWORD): cv.string,
-            vol.Optional(CONF_SSL, default=False): cv.boolean,
-            vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
-            vol.Optional(CONF_VERSION, default=DEFAULT_VERSION): vol.In([2, 3]),
-        }
-    )
-)
-
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We cannot fully remove this, instead use:

```py
CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Reply:
done